projects/sbb-esta/angular-business/styles.scss

```diff
@@ -1,3 +1,21 @@
 $sbbBusiness: true;
 
 @import '../angular-core/styles/_main.scss';
+
```

Comment:
This should be moved to the status.component.scss, either with ::ng-deep or with ViewEncapsulation.None